src/test/java/com/algorand/algosdk/transaction/TestTransaction.java

```diff
@@ -766,4 +768,127 @@ public void EmptyByteArraysShouldBeRejected() throws Exception {
         assertThat(tx.note).isNull();
         assertThat(tx.lease).isNull();
     }
+
+    @Nested
```

Comment:
This is the first time I've never seen this feature, nice way to group similar tests! :+1: